setup.cfg

```diff
@@ -71,23 +72,23 @@ dev =
     flake8-docstrings~=1.7.0
     pep8-naming~=0.13.3
     mypy~=1.5.1
-    pydocstyle~=6.3.0
+    #pydocstyle~=6.3.0
     coverage~=7.3.0
     pytest~=7.4.0
     pytest-cov~=4.1.0
     pytest-watch~=4.2.0
     pytest-xdist~=3.3.1
     pre-commit~=3.3.3
-    sphinx~=3.2.1
-    sphinx-rtd-theme~=0.5.0
+    sphinx~=7.3.7
```

Comment:
Might double-check this; the `github-actions-base` may force an older version of Sphinx to be used with the Roundup Action

Reply:
@nutjob4life hmmmm. we will see. it seems like there was some bug happening that was preventing me from building the docs locally. we will see if it works in production.

Reply:
@jordanpadams sounds good, fingers crossed 🤞